Assets/HoloToolkit/BuildAndDeploy/Editor/BuildDeployTools.cs

```diff
@@ -85,12 +85,22 @@ public static bool BuildSLN(string buildDirectory, bool showDialog = true)
                 WSAUwpSdk = EditorUserBuildSettings.wsaUWPSDK,
 
                 // Configure a post build action that will compile the generated solution
+                // Configure a post build action that will compile the generated solution
```

Comment:
This line looks like a duplicate... Is it just Gits's diff?

Reply:
I'll double check

Reply:
nope it was a dup.